data/events/scripts/player.lua

```diff
@@ -86,6 +86,25 @@ function Player:onLookInShop(itemType, count)
 end
 
 function Player:onMoveItem(item, count, fromPosition, toPosition, fromCylinder, toCylinder)
+	-- Store Inbox
```

Comment:
It may but we shouldn't care about that when it comes to new things, as I said a 1.2 datapack isent supposed to work with any other version, if it does great but we already created compat issues with 1.2 so there is no point in keeping things back when they anyways have to do updates.